ci-operator/step-registry/code-ready/snc/e2e/test/code-ready-snc-e2e-test-commands.sh

```diff
@@ -34,29 +34,7 @@ function run-tests() {
   pushd snc
   set -e
   export OPENSHIFT_PULL_SECRET_PATH="${HOME}"/pull-secret
-  ./snc.sh
-  if [[ $? -ne 0 ]]; then
-    exit 1
-  fi
-  # wait till the cluster is stable
-  sleep 5m
-  export KUBECONFIG=crc-tmp-install-data/auth/kubeconfig
-  # Remove all the failed Pods
-  oc delete pods --field-selector=status.phase=Failed -A
-
-  # Wait till all the pods are either running or pending or completed or in terminating state
-  while oc get pod --no-headers --all-namespaces | grep -v Running | grep -v Completed | grep -v Terminating | grep -v Pending; do
-     sleep 2
-  done
-
-  # Check the cluster operator output, status for available should be true for all operators
-  while oc get co -ojsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}' | grep -v True; do
-     sleep 2
-  done
-
-  # Run createdisk script
-  export SNC_VALIDATE_CERT=false
-  ./createdisk.sh crc-tmp-install-data
+  ./ci.sh
```

Comment:
Don't we need to keep 
```
  if [[ $? -ne 0 ]]; then
    exit 1
  fi
```
?

Reply:
does `set -e` cover us here ? I guess it should but we probably miss `set -e` in ci.sh.

Reply:
Ah right should be good enough. snc.sh uses something more extensive (`set -exuo pipefail`), but `-e` seems like what we need here.